### PR TITLE
Added String class rename to the list

### PR DESCRIPTION
**Pull request by jrbasso**
Related with cakephp/cakephp#5550

### CODE REVIEW

File: tests/test_files/RenameStringAfter.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?php
+
+use \Cake\Utility\Hash;
+use \Cake\Utility\Text;
```
Comment (dereuromark) [inline tests/test_files/RenameStringAfter.php]: no leading \ usually. Not that it would matter :)

**Comment by jippi** [comment]
wonder if the String => Text thing could be in done in 2.x too - would make hhvm possible :)

**Comment by dereuromark** [comment]
@jippi Using `String extends CakeText` that should be fairly easy.

File: tests/test_files/RenameStringAfter.php (hunk 1 above)
Comment (jrbasso) [inline tests/test_files/RenameStringAfter.php]: @dereuromark Removed for consistency.

**Comment by markstory** [comment]
Looks good to me.

**Pull request merged**